lib/data_update_scripts/20210831052422_add_connect_feature_flag.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class AddConnectFeatureFlag
+    def run
+      FeatureFlag.add(:connect)
```

Comment:
Should this only be applicable for DEV?

Reply:
No. This just adds the feature flag to the database, it does not enable it, that would be `FeatureFlag.enable(:connect)`.